hermes-frontend/src/main/java/pl/allegro/tech/hermes/frontend/producer/kafka/Producers.java

```diff
@@ -15,40 +15,42 @@
 import java.util.concurrent.atomic.AtomicBoolean;
 import java.util.function.Predicate;
 
+import static pl.allegro.tech.hermes.common.metric.Gauges.ACK_ALL;
+import static pl.allegro.tech.hermes.common.metric.Gauges.ACK_LEADER;
 import static pl.allegro.tech.hermes.common.metric.HermesMetrics.escapeDots;
 
 public class Producers {
-    private final Producer<byte[], byte[]> leaderConfirms;
-    private final Producer<byte[], byte[]> everyoneConfirms;
+    private final Producer<byte[], byte[]> ackLeader;
+    private final Producer<byte[], byte[]> ackAll;
 
     private final boolean reportNodeMetrics;
     private final AtomicBoolean nodeMetricsRegistered = new AtomicBoolean(false);
 
-    public Producers(Producer<byte[], byte[]> leaderConfirms,
-                     Producer<byte[], byte[]> everyoneConfirms,
+    public Producers(Producer<byte[], byte[]> ackLeader,
+                     Producer<byte[], byte[]> ackAll,
                      boolean reportNodeMetrics) {
-        this.leaderConfirms = leaderConfirms;
-        this.everyoneConfirms = everyoneConfirms;
+        this.ackLeader = ackLeader;
+        this.ackAll = ackAll;
         this.reportNodeMetrics = reportNodeMetrics;
     }
 
     public Producer<byte[], byte[]> get(Topic topic) {
-        return topic.isReplicationConfirmRequired() ? everyoneConfirms : leaderConfirms;
+        return topic.isReplicationConfirmRequired() ? ackAll : ackLeader;
     }
 
     public void registerGauges(HermesMetrics metrics) {
```

Comment:
Some time ago I started working on this redesign and to avoid copy-paste I cleaned up this method a bit: https://github.com/allegro/hermes/commit/7c9b95863ed6f1a09966ad9ddea3d68c82ad3f9b. If you think it is worth including my idea in this PR, feel free to use it.